### DISCUSSION TITLE
Please fix this source🙏🏻

### DISCUSSION THREAD

**Discussion by AimerEGOIST** [iptv-org/community#439] [Q&A] [answered]
![IMG_20230415_183117](https://user-images.githubusercontent.com/48597571/232210193-7bacee10-c91d-484b-a25d-bdf00501026e.jpg)

**Comment by AimerEGOIST** [comment]
Japanese AT-X Channel

**Comment by winnie3101** [comment]
What messages appearing?

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream
